test/extended/cluster/cl.go

```diff
@@ -198,6 +199,22 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 
 		// Wait for builds and deployments to complete
 		for _, ns := range namespaces {
+			podList, err := oc.AdminKubeClient().CoreV1().Pods(ns).List(metav1.ListOptions{})
+			if err != nil {
+				e2e.Logf("Error listing pods: %v", err)
+			}
+			podCount := len(podList.Items)
+			if podCount > 0 {
+				e2e.Logf("Waiting for %d pods in %s", podCount, ns)
+				pods, err := exutil.WaitForPods(c.CoreV1().Pods(ns), exutil.ParseLabelsOrDie(mapToString(podLabels)), exutil.CheckPodIsRunning, podCount, checkPodRunningTimeout)
+				if err != nil {
+					e2e.Failf("Error in pod wait... %v", err)
+				} else if len(pods) < podCount {
+					e2e.Failf("Only got %v out of %v pods", len(pods), podCount)
```

Comment:
Would it be possible to add a time formatting function to say something like 
`Only got X out of Y pods in FORMATTED_TIME`?  I believe that would help.